google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -263,6 +271,20 @@
 	    "allowedValues": "'((\\d{1,20})|(LATEST)|())'",
 	    "converterName": "ClientSideStatementValueConverters$StringValueConverter"
 	  }
+	},
+	{
+		"name": "SET OPTIMIZER_STATISTICS_PACKAGE = '<package>'|''",
+		"executorName": "ClientSideStatementSetExecutor",
+		"resultType": "NO_RESULT",
+		"regex": "(?is)\\A\\s*set\\s+optimizer_statistics_package\\s*(?:=)\\s*(.*)\\z",
+		"method": "statementSetOptimizerStatisticsPackage",
+		"exampleStatements": ["set optimizer_statistics_package='auto_20191128_14_47_22UTC'", "set optimizer_statistics_package=''"],
+		"setStatement": {
+			"propertyName": "OPTIMIZER_STATISTICS_PACKAGE",
+			"separator": "=",
+			"allowedValues": "'(([A-Za-z0-9\\-\\_]+)|())'",
```

Comment:
- [ ] Check if this regex is the correct one for the optimizer statistics package.